tests/suites/deploy/bundles/telegraf_bundle.yaml

```diff
@@ -1,21 +1,21 @@
-series: focal  # charms do not support jammy. 
+series: focal
 applications:
```

Comment:
Remove the comment because the test does a diff check with the bundle export output.